### PR TITLE
Added Stunner artifacts into KIE BOM.

### PR DESCRIPTION
**Pull request by romartin**
This is related to Stunner ( jbpm designer NEXT ) integration into kie-wb-common.

### CODE REVIEW

**Comment by romartin** [comment]
See https://github.com/droolsjbpm/kie-wb-common/pull/493

File: kie-bom/pom.xml [hunk 1]
```diff
@@ -1301,6 +1301,312 @@
...
+        <groupId>org.kie.workbench.stunner</groupId>
+        <artifactId>kie-wb-common-stunner-forms-api</artifactId>
+        <version>${version.org.kie}</version>
+        <classifier>sources</classifier>
+      </dependency>
+
+      <dependency>
+        <groupId>org.kie.workbench.stunner</groupId>
+        <artifactId>kie-wb-common-stunner-forms-backend</artifactId>
+        <version>${version.org.kie}</version>
```
Comment (manstis) [inline kie-bom/pom.xml]: Do we not also need ```kie-wb-common-stunner-forms-backend``` with ```sources``` classifier?

Comment (romartin) [inline kie-bom/pom.xml]: good point.. adding it :)

**Comment by romartin** [comment]
Closing this PR as I will provide a new one right one with the same branch name as the PR for kie-wb-common, so Jenkins should build all that stuff.

**Comment by romartin** [comment]
Reopened -> https://github.com/droolsjbpm/droolsjbpm-build-bootstrap/pull/290

**Pull request closed without merging**